src/rules/pydocstyle/rules/multi_line_summary_start.rs

```diff
@@ -27,21 +30,84 @@ pub fn multi_line_summary_start(checker: &mut Checker, docstring: &Docstring) {
+                for line in content_lines {
+                    if let Some(end_column) = line.find(|c: char| !c.is_whitespace()) {
+                        let start =
+                            Location::new(location.row(), location.column() + first_line.len());
+                        let end = Location::new(end_row, end_column);
+                        diagnostic.amend(Fix::deletion(start, end));
+                        break;
+                    }
+                    end_row += 1;
+                }
+            }
+            checker.diagnostics.push(diagnostic);
         }
     } else {
         if checker
             .settings
             .rules
             .enabled(&Rule::MultiLineSummarySecondLine)
         {
-            checker.diagnostics.push(Diagnostic::new(
+            let mut diagnostic = Diagnostic::new(
                 violations::MultiLineSummarySecondLine,
                 Range::from_located(docstring.expr),
-            ));
+            );
+            if checker.patch(diagnostic.kind.rule()) {
+                let mut indentation = String::from(docstring.indentation);
+                let mut fixable = true;
+                if !indentation.chars().all(char::is_whitespace) {
+                    fixable = false;
+
+                    // If the docstring isn't on its own line, look at the parent indentation, and
+                    // add the default indentation to get the "right" level.
+                    if let DefinitionKind::Class(parent)
+                    | DefinitionKind::NestedClass(parent)
+                    | DefinitionKind::Function(parent)
+                    | DefinitionKind::NestedFunction(parent)
+                    | DefinitionKind::Method(parent) = &docstring.kind
+                    {
+                        let parent_indentation =
+                            checker.locator.slice_source_code_range(&Range::new(
+                                Location::new(parent.location.row(), 0),
+                                Location::new(parent.location.row(), parent.location.column()),
+                            ));
+                        if parent_indentation.chars().all(char::is_whitespace) {
+                            indentation.clear();
+                            indentation.push_str(parent_indentation);
+                            indentation.push_str(checker.stylist.indentation());
+                            fixable = true;
+                        }
+                    };
+                }
+
+                if fixable {
+                    let location = docstring.expr.location;
+                    let prefix = leading_quote(contents).unwrap();
+                    // Use replacement instead of insert to trim possible whitespace between leading
+                    // quote and text.
+                    let repl = format!(
+                        "{}{}{}",
+                        checker.stylist.line_ending().as_str(),
```

Comment:
One minor code change -- need to be sensitive to the current line-ending style (could be `\r\n`, etc.).

Reply:
Ah, I thought about that and remember grepping for what should be done about it, but failed to follow through. Thanks for taking care of it!

There's a literal `\n` at https://github.com/charliermarsh/ruff/blob/662e29b1ce9dd99e1456aae6f5817335ca1a7562/src/autofix/helpers.rs#L182, too. Don't know if it's an issue, but thought I'd mention now that I noticed when re-grepping.

Reply:
Oh, thanks, I'll fix that real quick.